.gitlab-ci.yml

```diff
@@ -974,7 +974,7 @@ puppet-release:
   except:
     - schedules
   before_script:
-    - gem install bundler
+    - gem install bundler -v 2.4.22
```

Comment:
`gem install bundler -v '<2.5'` should be adequate. ruby 2.6 has been eol since 4/22 so any further investment beyond unblocking builds could include moving to ~3.1 or so as puppet support dictates.

Reply:
This very narrowly fixes the build and we need a followup. I’ll file a bug.